src/GraphQL/Type/PimcoreObjectType.php

```diff
@@ -95,11 +101,17 @@ public function build($context = [])
                         continue;
                     }
 
-                    /** @var $fieldHelper DataObjectFieldHelper */
-                    $fieldHelper = $this->getGraphQlService()->getObjectFieldHelper();
-                    $result = $fieldHelper->getQueryFieldConfigFromConfig($column, $class);
-                    if ($result) {
-                        $fields[$result['key']] = $result['config'];
+
+                    if (!$column["isOperator"] && is_array($column["attributes"]) && $column["attributes"]["dataType"] == "fieldcollections") {
+                        $this->addFieldCollectionDefs($column, $class, $fields);
+
+                    } else {
+                        /** @var $fieldHelper DataObjectFieldHelper */
+                        $fieldHelper = $this->getGraphQlService()->getObjectFieldHelper();
+                        $result = $fieldHelper->getQueryFieldConfigFromConfig($column, $class);
```

Comment:
`getQueryFieldConfigFromConfig` can return `bool|array`, what happens with `$fields[$result['key']]` if `$result` is `true` will it throw?